package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "captain-stack",
 	"displayName": "Captain Stack",
 	"description": "Find snippets codes from Stackoverflow",
-	"version": "0.0.5",
+	"version": "0.0.7",
```

Comment:
even i dont know why we are still on version 0.0.x just change it to 1.1.1 after this change.